fact/arc/arc.go

```diff
@@ -22,6 +22,11 @@ func (h Source) Facts() (map[string]interface{}, error) {
 
 	facts["arc_version"] = version.String()
 
+	// set online true when updating the facts
+	// this should fix the problem when deploying the API and the broker at the same time and the agents
+	// send the "online" message before the API is ready to accept incoming broker messages.
+	facts["online"] = true
```

Comment:
I'm note sure that this is affected. each plugin only generates a registration message when a value has changed. putting the value in the arc plugin will not cause it to send out a message after the first one.

Reply:
You are richt the arc source do not even have an interval set...
https://github.com/sapcc/arc/blob/master/server/server.go#L158